src/utils/jwt.go

```diff
@@ -7,12 +7,18 @@ import (
 	"gopkg.in/square/go-jose.v2/jwt"
 )
 
+type Features []string
+
 type JWTClaims struct {
 	Audience []string `json:"aud"`
+	Features Features `json:"1password.com/fts"`
 }
 
 const AudienceDEPRECATED = "com.1password.streamingservice"
 
+const ItemUsageFeatureScope = "itemusages"
```

Comment:
This const is duplicating the one at `main.ItemUsageFeatureScope`.  
Given the values are the same, can we drop the one at `main` package and use this one instead?